doc/checklist_new_release.md

```diff
@@ -1,51 +1,44 @@
-The following steps should be performed before releasing a new storm version.
-Note that in most cases a simultaneous release of [carl](https://github.com/moves-rwth/carl-storm), [storm](https://github.com/moves-rwth/storm), [pycarl](https://github.com/moves-rwth/pycarl/) and [stormpy](https://github.com/moves-rwth/stormpy/) is preferred.
+The following steps should be performed before releasing a new Storm version.
+Note that in most cases a simultaneous release of [carl-storm](https://github.com/moves-rwth/carl-storm), [Storm](https://github.com/moves-rwth/storm) and [stormpy](https://github.com/moves-rwth/stormpy/) is preferred.
 
 1. Update `CHANGELOG.md`:
-   * To get all the commits from an author since the last tag execute:
-   ```console
-   git log last_tag..HEAD --author "author_name"
-   ```
    * Set release month
+   * Set major changes.
+     Use the [automatically generated release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes).
+     Alternatively, get all the commits since the `last_tag` by executing:
+     ```console
+     git log last_tag..HEAD
+     ```
 
 2. Update used carl version:
-   * Update `STORM_CARL_GIT_TAG` in `CMakeLists.txt`
-   * Update `carl_tag` in `Dockerfile` and `.github/workflows/Dockerfile.archlinux`
+   This should be automatically handled by a CI workflow triggered by new releases in carl-storm.
```

Comment:
I would argue that steps 2-4 should be done first, and "before starting a new release", then the changelog needs to be updated (from which point onwards no PRs should be merged anymore until we have done the release).